invenio_base/app.py

```diff
@@ -216,8 +218,8 @@ def base_app(import_name, instance_path=None, static_folder=None,
     try:
         if not os.path.exists(instance_path):
             os.makedirs(instance_path)
-    except Exception:  # pragma: no cover
-        pass
+    except Exception as ex:  # pragma: no cover
+        app.logger.exception(ex)
```

Comment:
Ah yes, but it should be a string message. The ``logger.exception`` automatically grabs the exception:
``app.logger.exception("Failed to create instance path")``
https://docs.python.org/2/library/logging.html#logging.Logger.exception